xapi-profiles-structure.md

```diff
@@ -171,9 +171,14 @@ All Concepts in a Profile MUST follow the rules of one of the subsections within
 
 
 <a name="verb-activity-attach"></a>
-### 7.1 Verbs, Activity Types, and Attachment Usage Types
+### 7.1 Verbs, Activity Types, Category Context Activities and Attachment Usage Types
 
-Verb, Activity Type, and Attachment Usage Type Concepts share the same properties. They're all Concepts that make sense to relate semantically to others of the same type, such as indicating one is a narrower form of another.
+Verb, Activity Type, Category Context Activities, and Attachment Usage Type Concepts share the same properties. They're all Concepts that make sense to relate 
+semantically to others of the same type, such as indicating one is a narrower form of another. Category Context Activities were not included in the previous version,
+but is expected that the Category Context Activity corresponding to the xAPI Profile would be a reserved term for that xAPI Profile, but would still be considered a
+concept.  Use of Category Context Activities also allows the "reserving" or "sub-profiling" where a full xAPI Profile may not be created, but the use of a "tag" is 
+needed within a profile.  A Profile Creator SHOULD include the Category Context Activity corresponding to the xAPI Profile, versioning as appropriate, in its list of 
+Concepts.
```

Comment:
I'm not sure if we want to add Category Context Activities as a concepts, activities are already concepts.